src/common/hist_util.cu

```diff
@@ -27,9 +27,11 @@ namespace common {
 
```

Comment:
@RAMitchell is doing sketching algorithm anyway, these code is becoming legacy sooner or later.  It's not very motivating to change them ...